app/views/candidate_interface/application_form/_review.html.erb

```diff
@@ -86,8 +86,12 @@
 
 <section class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-m govuk-!-font-size-27"><%= t('section_groups.work_experience') %></h2>
-  <h3 class="govuk-heading-m"><%= t('page_titles.work_history') %></h3>
-  <%= render(CandidateInterface::WorkHistoryReviewComponent.new(application_form: application_form, editable: editable, heading_level: 4, show_incomplete: !application_form.work_history_completed && editable, missing_error: missing_error)) %>
+  <% if FeatureFlag.active?(:restructured_work_history) && @application_form.feature_restructured_work_history %>
+    <%= render(CandidateInterface::RestructuredWorkHistory::ReviewComponent.new(application_form: application_form, editable: editable, heading_level: 4, show_incomplete: !application_form.work_history_completed && editable, missing_error: missing_error)) %>
+  <% else %>
+    <h3 class="govuk-heading-m"><%= t('page_titles.work_history') %></h3>
+    <%= render(CandidateInterface::WorkHistoryReviewComponent.new(application_form: application_form, editable: editable, heading_level: 4, show_incomplete: !application_form.work_history_completed && editable, missing_error: missing_error)) %>
```

Comment:
@davidgisbey The logic has been fixed so the restructured view will appear if `FeatureFlag.active?(:restructured_work_history)` returns true. I've also added the `@application_form.feature_restructured_work_history` as an additional requirement.

Reply:
👌 yea that's perfect